recipe/meta.yaml

```diff
@@ -1,17 +1,18 @@
-{% set version = "0.4.3" %}
+{% set name = "py_stringmatching" %}
+{% set version = "0.4.5" %}
 
 package:
-  name: py_stringmatching
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/p/py_stringmatching/py_stringmatching-{{ version }}.tar.gz
-  sha256: 921b9bb163b310df341c33823913204f7b0ca9f6a319f5b451408300bd378dee
+  url: https://pypi.io/packages/source/p/{{ name }}/py-stringmatching-{{ version }}.tar.gz
+  sha256: d97ba1b562ad98effc946687c5dab074c17a515114ab0dcf502202d5cec9f2b5
 
 build:
-  number: 1
-  skip: true  # [not x86_64]
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 0
+  #skip: true  # [not x86_64]
```

Comment:
```suggestion
```

Reply:
Yep, saw that waiting for the last build to complete before marking for review.  Then forgot to push the change.